modules/internal/ChapelSyncvar.chpl

```diff
@@ -33,7 +33,7 @@ module ChapelSyncvar {
 
   /************************************ | *************************************
   *                                                                           *
-  * The implementation of the user-facing sync/single types are exposed to    *
+  * The implementation of the user-facing sync types are exposed to    *
```

Comment:
Nitty, but the spacing on the `*` for this is intended to be in a straight line down the comment.